### ISSUE TITLE
MacGap 2

### ISSUE THREAD

**Issue by jeff-h** [MacGapProject/MacGap1#132]
Just a heads-up that MacGap 2 is very nearly in beta and should be considered ready to test.

https://github.com/MacGapProject/MacGap2

Check it out!

The documentation site at http://docs.macgap.com is incomplete at present, but is a focus for us at this point.

MacGap 1 will only see minimal further development from this point on.

**Comment by MikeRich88** [comment]
macgap (dot) com now appears to be a phishing/malware site

**Comment by jeff-h** [comment]
Yep. Not much we can do about that, other than ensure we have no references to it in this project.
